pennychat/tests/api/test_penny_chat.py

```diff
@@ -38,13 +77,34 @@ def test_penny_chat_participants_list(test_chats_1):
         assert int(user_id) in [participant['user']['id'] for participant in chat['participants']]
 
 
+@pytest.mark.django_db
+def test_penny_chat_participants_list__other_content():
+    objects = setup_test_chats()
+    chat_private_1, chat_private_2 = objects['penny_chats']
+    user1, user2, user3 = objects['users']
+    soc_prof1, soc_prof2, soc_prof3 = objects['social_profiles']
+    client = APIClient()
+    token = Token.objects.create(user=user3)
+    client.credentials(HTTP_AUTHORIZATION='Token ' + token.key)
+    user_id = user2.id
+    response = client.get(f'/api/chats/?participants__user_id={user_id}')
+    assert response.status_code == 200
+    assert response.data['count'] == 0
+    chats = response.data['results']
+    for chat in chats:
+        assert int(user_id) in [participant['user']['id'] for participant in chat['participants']]
```

Comment:
Similar last test, you need extra test to prove that things that should be present are present and things that should not be present are not.

These 3 test cases: If I am looking at your profile
* I should see any public chat even if I didn't participate (this means you need to create a third public chat).
* I should see any private chat that you had where I was a participant.
* I should not see any private that that you participated in.

One way to make this test easier to compare to the last one is to look at the same profile page for both:

```
client.get(f'/api/chats/?participants__user_id={user1.id}')
```

And then it will be obvious what the assertions mean.